docs/references/react/use-user.mdx

```diff
@@ -78,6 +82,8 @@ export default function Home() {
 
 To retrieve the latest user data after updating the user elsewhere, use the `user.reload()` method.
 
+For more information on the `reload()` method, see the [`User`](/docs/references/javascript/user/user) reference.
```

Comment:
Reload is not documented on this page. I could only find it here: https://clerk.com/docs/references/react/use-user#reload-user-data-with-the-use-user-hook. We should add it to this reference. I've [created a Linear issue](https://linear.app/clerk/issue/DOCS-5856/add-reload-to-the-user-js-sdk-reference-page)

Reply:
oh you know what - because reload is only available on the User resource return from useUser . its not available on the User object... hmmm